src/main/zlatopolsky/chapter4/Ch1N4_15.java

```diff
@@ -0,0 +1,9 @@
+package zlatopolsky.chapter4;
+
+public class Ch1N4_15 {
+
+    public int humanAge(int yearOfBirth, int monthOfBirth, int yearCurrent, int monthCurrent) {
+        return ((yearCurrent * 12 +
+                monthCurrent) - (yearOfBirth * 12 + monthOfBirth)) / 12;
```

Comment:
// You need delete 12, because you multiply and divide  by 12

Reply:
![test AgeHuman](https://github.com/na206851/java-basics/assets/150899065/2759bde2-e0bb-478a-b05c-c427c8efd033)
no, i test your option, it produced uncorrected information